libsql-sqlite3/src/vectordiskann.c

```diff
@@ -64,6 +65,11 @@
 // stack simplify memory managment code and also doesn't impose very strict limits here since 128 bytes for column names should be enough for almost all use cases
 #define DISKANN_SQL_RENDER_LIMIT 128
 
+// limit to the maximum size of DiskANN block (128 MB)
+// even with 1MB we can store tens of thousands of nodes in several GBs - which is already too much
+// but we are "generous" here and allow user to store up to 128MB blobs
+#define DISKANN_MAX_BLOCK_SZ 134217728
```

Comment:
I wonder if anything actually works with block size this big? I think 1 MB is probably a reasonable limit.

Reply:
`128MB` is just pretty rough and loose upper bound. 
I don't want to limit blocks to 1MB because 1MB + eps (2MB for example) also can be usable. 
But `128MB` is just insane size for single node which no-one should actually use.